hbase-client/src/main/java/org/apache/omid/transaction/TTable.java

```diff
@@ -76,6 +71,10 @@
 
     private HTableInterface table;
 
+    private SnapshotFilter snapshotFilter;
+
+    private boolean serverSideFilter;
```

Comment:
No need to be a class field, you use it once in a function so make it local there.